### PR TITLE
Handle unknown and default Durations

### PR DESCRIPTION
**Pull request by raman325**
fixes https://github.com/home-assistant/core/issues/99873

### CODE REVIEW

**Review by MartinHjelmare — COMMENTED** [review]
Black complains.

**Comment by raman325** [comment]
> Black complains.

thanks, missed this

File: zwave_js_server/model/duration.py [hunk 1]
```diff
@@ -17,12 +17,16 @@ class DurationDataType(TypedDict, total=False):
 class Duration:
     """Duration class."""
 
-    data: DurationDataType
+    data: DurationDataType | Literal["unknown", "default"]
```
Comment (MartinHjelmare) [inline zwave_js_server/model/duration.py:20]: Does this mean that the unit in `DurationDataType` can only have seconds or minutes as values?

Comment (raman325) [reply zwave_js_server/model/duration.py:20]: yup good call

Comment (raman325) [reply zwave_js_server/model/duration.py:20]: reference: https://github.com/zwave-js/node-zwave-js/blob/master/packages/core/src/values/Duration.ts#L134

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**